Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -9,7 +9,6 @@ Due to package verification, the latest version below is the unpublished version
 however, it has to be formatted properly to pass verification tests.
 
 ## [Unreleased] - yyyy-mm-dd
-- Fixed InputControl picker not updating correctly when the Input Actions Window was dirty. [ISXB-1221](https://issuetracker.unity3d.com/product/unity/issues/guid/ISXB-1221)
```

Comment:
I think my brain left the building when I filled that up, it should have totally been in another section. Moving now.